proposals/PROPOSALS.md

```diff
@@ -0,0 +1,156 @@
+
+### The HF Technical Track
+
+The HF Technical Track is an experienced group of people with knowledge of the Haskell ecosystem, responsible for the strategic technical direction on behalf of the Haskell Foundation. Members are tasked with
+
+  - communicating with the community
+  - weighing in pros and cons of every proposal
+  - accepting, postponing or rejecting the proposal.
+
+HF Technical Track members should be either individuals responsible for a specific part of the Haskell ecosystem, or contributors and committers to parts of it. The members are selected by the HF CTO based on their expertise and implication in the community.
+
+The current HF Technical Track members are:
+
+- Emily Pillmore ([@emilypi](https://github.com/emilypi)), Haskell Foundation
+- Richard Eisenberg ([@goldfirere](https://github.com/goldfirere)), Tweag
+- Michael Snoyman ([@snoyberg](https://github.com/snoyberg)), FPComplete
+- Andrew Lelechenko ([@Bodigrim](https://github.com/Bodigrim)), Barclays
+- Davean Scies ([@davean](https://github.com/davean)), XKCD
+- Edward Kmett ([@ekmett](https://github.com/ekmett)), MIRI
+- Theophile Choutri ([@Kleidukos](https://github.com/Kleidukos)), Scrive
+- Gil Mizrahi (@soupi)
+
+### Voting
+
+When a HIP is scheduled at an HF Technical Track meeting (i.e. it is in 'Under Review'), it can be held to a vote for one of the following outcomes (in case it makes a difference: the HIP will be marked according to the first on this list to have a majority):
+
+- Accepted (needs 66% of the HF Technical Track to vote in favor, and the HIP must specify a proposal lead, e.g. an implementor or project director, from the HF Technical Track who will represent the work done on the project.)
+- Dormant (needs a simple majority, an HF Technical Track member that voted in favor will close the issue and PR and mark it as Dormant)
+- Postponed (needs a simple majority, an HF Technical Track member that voted to postpone will close the issue and PR, and write clear conditions for reopening)
+- Revision needed (needs a simple majority. This can only be the outcome of the vote four times, for a total of five rounds. An HF Technical Track member will write up what revisions are needed.)
+
+If none of these applies, the proposal is Rejected.
+
+### Responsibilities of the members
+
+- Review all HIPs 
+- Play a role in the discussions, learn in advance about the topic if needed, and make up their mind in the voting process.
+
+### Guests
+
+Experts in some fields may be invited to specific meetings as guests when discussing related HIPs. Their input would be important to discuss the current state of the proposal, both its design and implementation.
+
+## Proposal states
```

Comment:
How can I see the state of my proposal?  How can I see the state of all proposals?  Maybe no need to answer here, but once in operation these should be easy questions to answer.

Reply:
Good point. I'll add a sentence about using the Github Label system